docs/eigenlayer/restaking-guides/0-restaking-user-guide/native-restaking/withdraw-from-eigenlayer/full-withdrawal.md

```diff
@@ -4,41 +4,34 @@ sidebar_position: 2
 
 # Full Withdrawal
 
-Full Withdrawal is the process of unlocking the entire balance held by their validator, including both the initial stake and any accumulated rewards.
+Full Withdrawal is the process of unlocking the entire balance held by their validator, including both the initial stake and any accumulated rewards. The following order of operations are required to complete a Full withdrawal of your natively restaked ETH and avoid proof generation errors.
 
-All funds unstaked from EigenLayer go through a escrow period before being able to be withdrawn. After you unstake and queue the withdrawal, you must wait for the escrow period to end before being able to withdraw your assets.
 
-:::tip
-Exits from the beacon chain can be completed in independently of (in parallel, at the same time as) queuing a Full Withdrawal from EigenLayer.  This optimization may reduce your overall (end to end) time required to withdraw Native restaked funds from EigenLayer.
-:::
+**Step 1:** Ensure you have [repointed your validator's withdrawal credentials](../create-eigenpod-and-set-withdrawal-credentials/repointing-a-validators-withdrawal-credentials.md) to your EigenPod.
 
-:::warning
-Ensure you have [repointed your withdrawal credentials](../create-eigenpod-and-set-withdrawal-credentials/repointing-a-validators-withdrawal-credentials.md) to your EigenPod before proceeding.
-:::
+**Step 2:** Fully exit your validator from the beacon chain. You may monitor its activity via beaconcha.in/validator/[yourvalidatorid] .
```

Comment:
nit: [yourvalidatorindex] or [yourvalidatorpublickey]

Reply:
Thanks, that's a good nit